src/Makefile.gtest.include

```diff
@@ -1,3 +1,4 @@
+TESTS += zcash-gtest
```

Comment:
ACK w/ caveat:

I believe this means it is impossible to get coverage information about *only* ``test_bitcoin``, and instead it will be the union of ``test_bitcoin`` and ``zcash-gtest``. I may be wrong. Anyway, since the primary coverage information I care about is ``zcash-gtest`` and ``make cov-zcash`` should record only that, I'm fine with this change.

Plus it seems more consistent in terms of ``make check`` running all unit tests.